packages/astro/src/cli/add/index.ts

```diff
@@ -183,9 +201,32 @@ export async function add(names: string[], { flags }: AddOptions) {
 					integrationName: 'Svelte',
 					possibleConfigFiles: ['./svelte.config.js', './svelte.config.cjs', './svelte.config.mjs'],
 					defaultConfigFile: './svelte.config.js',
-					defaultConfigContent: SVELTE_CONFIG_STUB,
+					defaultConfigContent: STUBS.SVELTE_CONFIG,
 				});
 			}
+			if (integrations.find((integration) => integration.id === 'db')) {
+				if (!existsSync(new URL('./db/', root))) {
+					logger.info(
+						'SKIP_FORMAT',
+						`\n  ${magenta(`Astro will scaffold ${green('./db/config.ts')}${magenta(' and ')}${green('./db/seed.ts')}${magenta(' files.')}`)}\n`
+					);
+
+					if (await askToContinue({ flags })) {
+						await fs.mkdir(new URL('./db/', root), { recursive: true });
```

Comment:
nit: `recursive` isn't needed because we already checked if the folder doesn't exist